document_meta.go

```diff
@@ -357,6 +357,21 @@ func fieldName(sf reflect.StructField) (string, bool) {
 	return snaker.CamelToSnake(sf.Name), false
 }
 
+func isEmbedded(sf reflect.StructField) bool {
+	// anonymous structs are always embedded
```

Comment:
Is this still true to provide backwards compatibility?

Reply:
anonymous struct is basically an embedded struct, so I think no problem